### PR TITLE
Add contact section and footer

### PR DESCRIPTION
**Pull request by Kamba56**
Added a contact section to the Markup
Added styling to the About section
Added footer and style it
Added screenshot of the section to the README.md file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -251,6 +251,58 @@ <h3 class="semi-bold center-text work-title">Skills</h3>
...
+            type="email"
+            name="email"
+            placeholder="Email address"
+            required
+          />
+          <textarea name="comment" id="comment" cols="30" rows="10" required>
+Enter text here</textarea
+          >
+          <button type="submit" class="btn">Get in touch</button>
+        </form>
```
Comment (aimemalaika) [inline index.html]: - [ ] Kindly submit your form to [formspree service](https://formspree.io/html) as stated in the project requirements

File: index.html [hunk 2]
```diff
@@ -251,6 +251,58 @@ <h3 class="semi-bold center-text work-title">Skills</h3>
...
+        >
+          <input type="text" name="name" placeholder="Full name" required />
+          <input
+            type="email"
+            name="email"
+            placeholder="Email address"
+            required
+          />
+          <textarea name="comment" id="comment" cols="30" rows="10" required>
+Enter text here</textarea
```
Comment (aimemalaika) [inline index.html]: - [ ] Kindly make sure the message input has a max length of 500 characters
- [ ] Kindly make sure the text area is required

File: index.html [hunk 3]
```diff
@@ -251,6 +251,58 @@ <h3 class="semi-bold center-text work-title">Skills</h3>
...
+        </h2>
+
+        <form
+          action="#"
+          method="post"
+          id="contact-form"
+          class="flex-column center-align"
+        >
+          <input type="text" name="name" placeholder="Full name" required />
+          <input
```
Comment (aimemalaika) [inline index.html]: - [ ] Kindly make sure the name input has a max length of 30

**Review by aimemalaika — CHANGES_REQUESTED** [review]
Hi @Kamba56, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

the-simpsons-homer-simpson

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (Kamba56) [reply index.html]: but i already have a required attribute on my textarea @aimemalaika

File: index.html [hunk 4]
```diff
@@ -251,6 +251,58 @@ <h3 class="semi-bold center-text work-title">Skills</h3>
...
+          <input type="text" name="name" placeholder="Full name" maxlength="30" required>
+          <input
+            type="email"
+            name="email"
+            placeholder="Email address"
+            required
+          />
+          <textarea name="message" id="comment" cols="30" rows="10" maxlength="500" required>
+Enter text here</textarea
+          >
```
Comment (williamrolando88) [inline index.html:276]: - [ ] [Optional] Is a good practice to add helping text in your form inside a `placeholder` attribute, for example for this case you should use
``` html
<textarea name="message" .... placeholder="Enter text here" required></textarea>
```
I strongly suggest you add this change to your form, to improve the user experience.

image

- [ ] [Optional] Also if you look at your textarea you can also add a vertical padding, to improve the looks of your form

**Review by williamrolando88 — APPROVED** [review]
Hi @Kamba56 , 

## Your project is complete!
There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

hurra!

## Highlights

- Nice work with your project, looks very professional and clean

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 4 above)
Comment (Kamba56) [reply index.html:276]: Thank you @williamrolando88

**Pull request merged**